### PR TITLE
Ensure this package works on osx windows and linux

### PR DESCRIPTION
**Pull request by JakeChampion**
Currently it is failing on osx due to an incompatible dependency, this was missed because our automated testing only ran on linux.

This change makes the tests run on windows, osx and linux.

I've changed the dependency to one which does work on windows, osx and linux.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     "test": "make verify test"
   },
   "dependencies": {
+    "@financial-times/disk-space": "^1.0.0",
```
Comment (notlee) [inline package.json:24]: What this doesn't exist?!

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+          - windows-latest
+        node_version:
+          - 10
+          - 12
+          - 14
+        architecture:
+          - x64
+        # an extra windows-x86 run:
+        include:
+          - os: windows-2016
```
Comment (notlee) [inline .github/workflows/test.yml:24]: This run is maybe overkill? But excellent work. Thank you for tacking down the issue _and_ improving the workflow to test the operating systems we care about 😊

**Review by notlee — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (JakeChampion) [reply package.json:24]: I forked the disk-space package and made a version which works on windows and osx as well as linux

File: .github/workflows/test.yml (hunk 2 above)
Comment (JakeChampion) [reply .github/workflows/test.yml:24]: @notlee shall I remove it?

Comment (JakeChampion) [reply .github/workflows/test.yml]: I've removed it 👍

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v2.1.9 :tada:

**Pull request merged**